intake/container/dataframe.py

```diff
@@ -51,6 +51,12 @@ def to_dask(self):
         self._load_metadata()
         return self.dataframe
 
+    def to_ibis(self):
+        import ibis.pandas
```

Comment:
These are cute, but perhaps not very useful. We can wrap an dataframe in an ibis client and perform lazy operations on them. It all happens in memory, however. I'm agnostic to whether they should actually be included here.

Reply:
I take it, there's no way to wrap a Dask lazy dataframe

Reply:
I don't think so. Or, at least, it is not designed to be able to do that.